tests/integration/targets/memorydb/aliases

```diff
@@ -1 +1,4 @@
 cloud/aws
+
+# Missing permissions
+disabled
```

Comment:
Will https://github.com/mattclay/aws-terminator/pull/291 not allow us to run these tests?

Reply:
I disabled the tests because at that time there wasn't a PR with the terminator policies. I assume (but have to test), tests will pass.